go/systests/ephemeral_test.go

```diff
@@ -370,6 +371,14 @@ func readdToTeamWithEKs(t *testing.T, leave bool) {
 	}
 
 	user1.addTeamMember(teamName.String(), user2.username, keybase1.TeamRole_WRITER)
+
+	maxGen, err := user1.tc.G.GetTeamEKBoxStorage().MaxGeneration(context.Background(), teamID)
```

Comment:
`MaxGeneration` is only a local call and doesn't fetch what is on the server, for these tests it's better to check that the generation increased as you expected and each user can successfully call `Get` which will first try the cache and then try the server

Reply:
ah makes sense, thank you!